crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -26,11 +26,19 @@ use std::{
 };
 
 use futures_util::stream::{self, StreamExt};
+pub use matrix_sdk_base::crypto::{
+    olm::{
+        SessionCreationError as MegolmSessionCreationError,
+        SessionExportError as OlmSessionExportError,
+    },
+    CryptoStoreError, DecodeError, DecryptorError, Ed25519SignatureError, EventError, KeyError,
+    KeyExportError, LocalTrust, MediaEncryptionInfo, MegolmDecryptionError, MegolmError,
+    OlmDecryptionError, OlmError, OlmSessionCreationError, PickleError, RoomKeyImportResult,
+    SecretImportError, SessionCreationError, SessionKeyDecodeError, SignatureError,
```

Comment:
That's a lot of error types :sweat_smile:

Reply:
Yes, I think half of those are the crypto crate while the other are from vodozemac. Would you prefer if they're separated in submodules?

Reply:
Hmm, that's a good idea. Yeah let's do that.

Reply:
I only created a module for the re-exports from vodozemac since it seemed to me to be enough of a split.